lenet/README.md

```diff
@@ -0,0 +1,55 @@
+## WebNN API LeNet Example
+This example showcases the [LeNet](http://yann.lecun.com/exdb/publis/pdf/lecun-01a.pdf)-based handwritten digits classification by WebNN API.
+
+This example leverages the network topology of [the LeNet example of Caffe*](https://github.com/BVLC/caffe/tree/master/examples/mnist), the weights of [the LeNet example of OpenVINO*](https://github.com/openvinotoolkit/openvino/tree/2020/inference-engine/samples/ngraph_function_creation_sample) and the MNIST dataset of [mnist.js](https://github.com/cazala/mnist).
+
+The nodes of the LeNet used in this example are listed in the following table.
+
+| Name | WebNN op | Remarks |
```

Comment:
can you add the inputs of each op.

Reply:
The model is sequential. I list these ops in topology order. So an op is the input to the op in next row. I can put that notes into the README.md. Does that work?

Reply:
I am thinking it might be better to draw a diagram. I'll come up with one for your review.

Reply:
I added the topology diagram into the README.md. I'am pasting it here for convenience..

![image](https://user-images.githubusercontent.com/1005673/93610654-ead49b00-f9ff-11ea-88e3-2160f7edf1bc.png)

Does it look good to you?

Reply:
After check with @pyu10055 , it would be better to use a higher level illustration. So I use the diagram of [lenet.prototxt](https://github.com/BVLC/caffe/blob/master/examples/mnist/lenet.prototxt) rendered by [netron](https://lutzroeder.github.io/netron/).

![image](https://user-images.githubusercontent.com/1005673/93839040-3d42df80-fcbe-11ea-8136-f1bea1415a3e.png)

I also updated the table that maps the layer to webnn ops.

@pyu10055 , please take another look. Thanks!

Reply:
Thanks! This looks great!

Reply:
great, thanks for reviewing!